src/nvim/help.c

```diff
@@ -1119,18 +835,14 @@ void ex_helptags(exarg_T *eap)
     eap->arg = skipwhite(eap->arg + 3);
   }
 
-  if (strcmp(eap->arg, "ALL") == 0) {
-    do_in_path(p_rtp, "", "doc", DIP_ALL + DIP_DIR, helptags_cb, &add_help_tags);
-  } else {
-    ExpandInit(&xpc);
-    xpc.xp_context = EXPAND_DIRECTORIES;
-    char *dirname =
-      ExpandOne(&xpc, eap->arg, NULL, WILD_LIST_NOTFOUND|WILD_SILENT, WILD_EXPAND_FREE);
-    if (dirname == NULL || !os_isdir(dirname)) {
-      semsg(_("E150: Not a directory: %s"), eap->arg);
-    } else {
-      do_helptags(dirname, add_help_tags, false);
-    }
-    xfree(dirname);
+  MAXSIZE_TEMP_ARRAY(args, 2);
+  ADD_C(args, CSTR_AS_OBJ(eap->arg));
+  ADD_C(args, BOOLEAN_OBJ(add_help_tags));
+
+  Error err = ERROR_INIT;
+  // NLUA_EXEC_STATIC("require('vim._core.help').gen_tags(...)", args,
+  //                  kRetNilBool, NULL, &err);
```

Comment:
`module 'vim._core.help' not found`. Is this related to #36870?

Reply:
If it is called with `nvim -ll` or `nvim -u NONE` then probably.